lib/DwarfImporter/DwarfImporter.cpp

```diff
@@ -745,7 +746,7 @@ class DwarfToModelConverter {
     using namespace model;
 
     // Create function type
-    UpcastableType NewType = makeType<CABIFunctionType>();
+    UpcastablePointer<model::Type> NewType = makeType<CABIFunctionType>();
```

Comment:
Why?

Reply:
At some point it was not possible to have a `using UpcastableType = UpcastablePointer<model::Type>` in a place which made sense, so I expanded it manually. I'm not 100% sure what the problem was, but now it's solved